client/src/AppRoutes.js

```diff
@@ -30,6 +31,7 @@ const AppRoutes = () => (
       <Route path="/signup" element={<SignupPage />} />
       <Route path="/favorites" element={<FavoritesPage />} />
       <Route path="*" element={<ErrorPage />} />
+      <Route path="/contacts" element= {<ContactUsPage />} />
```

Comment:
mistake. please fix. Error page with path /* has always to be the last in the list of Routes.

Reply:
agreed. done